lib/slack/handlers.ex

```diff
@@ -72,6 +72,10 @@ defmodule Slack.Handlers do
     end
   end)
 
+  def handle_slack(%{type: "im_created", channel: channel}, slack) do
```

Comment:
Should we also handle `im_open` and `im_close`? If it's out of scope for this PR an issue is also okay.